src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -10,16 +11,32 @@ public class ReadWriteList<E> {
     private ReadWriteLock lock = new ReentrantReadWriteLock();
 
     public void add(E element) {
-        // write your code here
+        Lock writeLock = lock.writeLock();
```

Comment:
it's redundant to create a local variable each time method called, if you check implementation, there is only one instance of read and write locks